### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by DuskEcho** [lmmfranco/nintendo-switch-eshop#83]
**Please describe the problem you are having in as much detail as possible:**

getting: "EshopError: Fetching of US Games failed" on calling 
getGamesUS();
and
getGamesUS({shop: 'unfiltered'});

{ EshopError: Fetching of US Games failed
    at new e (...\node_modules\nintendo-switch-eshop\dist\index.js:1:156866)
    at ...\node_modules\nintendo-switch-eshop\dist\index.js:1:160490
    at ...\node_modules\nintendo-switch-eshop\dist\index.js:1:2156
    at Object.throw (...\node_modules\nintendo-switch-eshop\dist\index.js:1:2261)
    at o (...\node_modules\nintendo-switch-eshop\dist\index.js:1:1070)
    at process._tickCallback (internal/process/next_tick.js:68:7) name: 'EshopError' }

getGamesEurope() seems unaffected.

**Include a reproducible code sample here, if possible:**
```js

module.exports = {
    setAllGames: async () => {
        console.log("searching getGamesAmerica");
        allGames = await getGamesAmerica({shop: 'unfiltered'});
        console.log("Total games: " + allGames.length);
    },
}
// Place your code here

```

**Further details:**
- version: 2.2.4
- Node.js version:  10.16.3
- Operating system:  Windows 10 - 64 bit
- Priority this issue should have – please be realistic and elaborate if possible: Fairly high if getGamesUS is actually broken, fairly low if user error

- [ ] I have also tested the issue on latest master, commit hash:

**Comment by Favna** [comment]
Looks like Nintendo changed the limit from 1K games per request to 100 games per request. Fixed in master (83d89348cdec76483f7839f798289e6780b64c82) now, just waiting for all the CI to pass before I publish.

**Comment by Favna** [comment]
published v2.2.5 with the fixes to NPM
